### DISCUSSION TITLE
which dht library?

### DISCUSSION THREAD

**Discussion by kolczak** [GoWired/GoWired-Project#17] [Q&A] [answered]
Hi,
could you point out which library are you using for dht? When compiling IDE shows me following error:
```
dht.h: No such file or directory
```
When I try to include `DHT sensor library` library using top bar menu Arduino IDE adds:
```
#include <DHT.h>
#include <DHT_U.h>
```

**Comment by feanor-anglin** [comment, marked as answer]
Answered here https://github.com/feanor-anglin/GetWired-Project/issues/18
